conda-store-server/tests/plugins/test_plugin_context.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import logging
+import os
+import subprocess
+
+import pytest
+
+from conda_store_server.plugins.plugin_context import PluginContext
+
+
+class TestOutPutCapture:
+    def __init__(self):
+        self.output = ""
+
+    def write(self, line: str):
+        self.output += line
```

Comment:
Would it be better to use `io.StringIO` here? Not sure if we need a new class for this.